jiant/tasks/tasks.py

```diff
@@ -2380,6 +2428,12 @@ def __init__(self, path, max_seq_len, name, tokenizer_name, **kw):
         self.val_data_text = None
         self.test_data_text = None
 
+    def update_metrics(self, out, batch):
+        logits = out["logits"]
+        labels = batch["targs"]["words"][:, :seq_len].contiguous().view(-1)
```

Comment:
Looks like it'll break with unresolved reference ('seq_len').

Reply:
Ah, good catch!